src/patternfly/components/OverflowMenu/examples/index.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import Documentation from '@siteComponents/Documentation';
+import Example from '@siteComponents/Example';
+
+import overflowMenuSimpleExampleRaw from '!raw!./overflow-menu-simple-example.hbs';
+import overflowMenuGroupTypesExampleRaw from '!raw!./overflow-menu-group-types-example.hbs';
+import overflowMenuSimpleAdditionalOptionsExampleRaw from '!raw!./overflow-menu-simple-additional-options-example.hbs';
+import overflowMenuPersistentExampleRaw from '!raw!./overflow-menu-persistent-example.hbs';
+
+import OverflowMenuSimpleExample from './overflow-menu-simple-example.hbs';
+import OverflowMenuGroupTypesExample from './overflow-menu-group-types-example.hbs';
+import OverflowMenuSimpleAdditionalOptionsExample from './overflow-menu-simple-additional-options-example.hbs';
+import OverflowMenuPersistentExample from './overflow-menu-persistent-example.hbs';
+
+import overflowMenuSimpleExampleDoc from '../docs/overflow-menu-simple.md';
+import overflowMenuGroupTypesExampleDoc from '../docs/overflow-menu-group-types.md';
+import overflowMenuSimpleAdditionalOptionsExampleDoc from '../docs/overflow-menu-simple-additional-options.md';
+import overflowMenuPersistentExampleDoc from '../docs/overflow-menu-persistent.md';
+import overflowMenuIntroDoc from '../docs/overflow-menu-intro.md';
+
+import docs from '../docs/code.md';
+
+export const Docs = docs;
+
+export default props => {
+  const overflowMenuSimpleExample = OverflowMenuSimpleExample();
+  const overflowMenuGroupTypesExample = OverflowMenuGroupTypesExample();
+  const overflowMenuSimpleAdditionalOptionsExample = OverflowMenuSimpleAdditionalOptionsExample();
+  const overflowMenuPersistentExample = OverflowMenuPersistentExample();
+  const headingText = 'Overflow menu';
+  const variablesRoot = 'pf-c-overflow-menu';
+
+  return (
+    <Documentation data={props} docs={Docs} heading={headingText} variablesRoot={variablesRoot}>
+      <Example
+        intro={overflowMenuIntroDoc}
+        heading="Overflow menu simple (responsive)"
+        handlebars={overflowMenuSimpleExampleRaw}
+        docs={overflowMenuSimpleExampleDoc}
+        className="is-layout-page"
+      >
+        {overflowMenuSimpleExample}
+      </Example>
+      <Example
+        heading="Overflow menu group types"
```

Comment:
For this example,  I think you should have the kebab show at a larger breakpoint, right now its md, but I think we should show a better use of the breakpoint values so that it isn't off the page:

<img width="694" alt="Screen Shot 2019-09-04 at 5 48 26 PM" src="https://user-images.githubusercontent.com/20118816/64294292-6621f680-cf3c-11e9-950d-6c19fdb2f927.png">